docs/INSTALLATION.md

```diff
@@ -4,11 +4,22 @@
 
 Behind the scenes, this script downloads a binary from the latest GitHub release.
 
-**This will activate `sudo` to write the executable.** Please review the install script before
-running it.
+### Windows
+
+Run this command on PowerShell:
+```shell
+irm https://raw.githubusercontent.com/spenserblack/gengo/main/install.ps1 | iex
```

Comment:
Nitpick: I prefer the longer forms of the cmdlets with code that will most likely be copied and pasted.
```suggestion
Invoke-RestMethod https://raw.githubusercontent.com/spenserblack/gengo/main/install.ps1 | Invoke-Expression
```